clients/da-vinci-client/src/main/java/com/linkedin/davinci/repository/VeniceMetadataRepositoryBuilder.java

```diff
@@ -147,7 +147,6 @@ private void initServerStoreAndSchemaRepository() {
         adapter,
         clusterConfig.getRefreshAttemptsForZkReconnect(),
         clusterConfig.getRefreshIntervalForZkReconnectInMs());
-    storeConfigRepo.refresh();
```

Comment:
I think we should still call `refresh` to get a list of stores, and please see my following comments.

Reply:
Thanks for the great suggestions! The core idea is only place watch when there's a need to get store config, and for other stores, we just cache them to know whether they are present or not. Will update it